src/main/java/shedar/mods/ic2/nuclearcontrol/items/ItemCardMultipleSensorLocation.java

```diff
@@ -151,7 +151,7 @@ public CardState updateGenerator(TileEntity panel, ICardWrapper card, int range)
 		TileEntity entity = panel.getWorldObj().getTileEntity(target.posX, target.posY, target.posZ);
 		if (entity instanceof TileEntityBaseGenerator) {
 			// int production = ((TileEntityBaseGenerator)entity).production;
-			int production = (int) EnergyNet.instance.getTotalEnergyEmitted(entity);// TODO deprecated
+			int production = (int) EnergyNet.instance.getNodeStats(entity).getEnergyOut();

```

Comment:
It is the same. I comared the code from IC2 Exp for that and it is returning the same value so no worry